drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -17,6 +17,17 @@ matterManufacturer:
     vendorId: 0x1426
     productId: 0x0003
     deviceProfileName: plug-binary
+#Elko
+  - id: "5170/4096"
+    deviceLabel: RFSAI-62B-SL/MT
+    vendorId: 0x1432
+    productId: 0x1000
+    deviceProfileName: switch-2
```

Comment:
Matter-switch devices that are multi-component should use the `switch-binary` in the fingerprint, and then depend on the driver to create a child device for each additional switch. For example, this device would join with `switch-binary` with the parent device covering the main endpoint (in this case, the lowest numbered endpoint that supports the OnOff cluster), and then an additional child device would be made for the additional child endpoint. This device would use parent-child and appear as two device cards.

With the current state of the matter-switch driver, I would not expect this profile to work as expected by design. These profiles are kept around though so we still have support for switch devices that were onboarded before the `matter-switch` driver was transitioned to a primarily parent-child driver.

Reply:
I will update this PR to point to the `switch-binary` profile for this device.

Reply:
@ctowns 👍 Updated.